src/Service/AddressLookup/PdokService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tychovbh\Mvc\Service\AddressLookup;
+
+use GuzzleHttp\Client;
+use Illuminate\Support\Arr;
+
+class PdokService
+{
+    /**
+     * @param $address
+     * @param $zipcode
+     * @param $house_number
+     * @return mixed
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public static function search($address, $zipcode, $house_number)
```

Comment:
I dont want the $address here, this means I can only use this service when I have an Address $address. 
What if I dont have an $address but still want to make a search? Move this logic to where you use  it:
```php
$address = new Address();
$search = PdokService::search('2352cz', 38);
$address->streetname = $search['streetname'];
```
or even simplier:
```php
$address = new Address(PdokService::search('2352cz', 38))
// or
$address = new Address();
$address->fill(PdokService::search('2352cz', 38));
```
Make sure search() methode returns a clean array with the data you need:

```php
return [
    'streetname' => Arr::get($fullAddress, 'response.docs.0.straatnaam')
];
```

missing param typehinting and return type:
```php
search(string $zipcode, int $house_number): array
```